src/Uri.php

```diff
@@ -682,7 +682,7 @@ public function getFragment()
      * You can check if a scheme is valid before setting it using the
      * validateScheme() method.
      *
-     * @param  string $scheme
+     * @param  string|null $scheme
```

Comment:
The change here is ok, but in that occasion we should also check properties in that class and doc block there. 
I just checked `$scheme` and there we have also `string` only, but it should be `null|string`.

The same comment applied to all other changes in that file.